### ISSUE TITLE
 A11y_VisualStudioCodeClient_Integrated Terminal_keyboard:Keyboard users are unable to move focus to the list of split terminal windows

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141710]
GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev1; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #WCAG2.1.1; #Keyboard; #AINotInScope;

Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

Repro Steps:
1.Open Visual studio code editor.
2.Using "Ctrl+`" key open the Integrated Terminals.
3) Using  "Ctrl+\" key to split the terminal
4) Verify whether the list of split terminals are accessible using keyboard or not.

Actual Result:
Using Keyboard unable to access the list split terminal window.

Expected Result:
All the controls within the split terminal should be accessible using keyboard.

User Impact:
Keyboard users will face difficulty as the controls are not accessible using keyboard.
![#141710_A11y_VisualStudioCodeClient_Integrated Terminal_keyboard](https://user-images.githubusercontent.com/98530069/151556933-49ab6a54-944e-4ec9-82ad-e9ad7bc6841a.gif)

**Comment by meganrogge** [comment]
I believe what you're looking for is the command `workbench.action.terminal.focusTabs`?

**Comment by Tyriar** [comment]
You need to either focus via the command called out above, or run `toggle tab key moves focus` in order to use tab in the terminal. Unfortunately since tab is extremely important in the terminal this is how it has to work afaict.
